unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/ImpostorsTestPanel.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ImpostorsTestPanel : MonoBehaviour
```

Comment:
The same for this file.

Reply:
Removed Test panel, moved it into [another branch](https://github.com/decentraland/unity-renderer/pull/1111)